src/components/Facility/FacilityHome.tsx

```diff
@@ -106,17 +103,6 @@ export const FacilityHome = ({ facilityId }: Props) => {
       pathParams: { id: facilityId },
     }),
   });
-  const { mutate: deleteFacility } = useMutation({
```

Comment:
Let's not remove the deleteFacility logic. Move the confirm dialog to Dialog, and hide the option for now.